.github/workflows/test.yml

```
@@ -322,4 +315,68 @@ jobs:
+          echo "status=$STATUS" >> $GITHUB_OUTPUT
+
+      - name: Comment on PR if EOL warning
+        if: github.event_name == 'pull_request' && steps.parse.outputs.status == 'WARNING'
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const output = `${{ steps.eol.outputs.output }}`;
+
+            const body = `
+
+  \`\`\`
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)